### DISCUSSION TITLE
Issue with roles in Ansible

### DISCUSSION THREAD

**Discussion by Sreeni002** [learndevopsonline/learndevopsonline#1049] [Q&A] [answered]
HI Raghu,

I started doing practice on Ansible and getting error below message on roles. I did follow the same structure as followed and I tried my best to fix like earlier issue in Terraforms. Somewhere I am missing the folder structure or something else. Could you please help me to understand where I am doing mistake. PFB screenshot for your reference

![image](https://github.com/learndevopsonline/learndevopsonline/assets/129163655/9478022b-366a-4eb2-abd7-45e012ec8f05)

**Comment by learninguser** [comment, marked as answer]
Hi,

The command that your executing for running your playbook is incorrect. It should be `ansible_user=centos` and `ansible_password=DevOps321` but not `ansible-username` and `ansible-password`. 

Kindly correct this and try to rerun the command. If the issue still persists, please reply on this thread. 

If its solved, kindly mark it as resolved

**Comment by Sreeni002** [comment]
Thank you much for the correcrtion
